sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -72,16 +60,16 @@ def lambda_handler(event, context):
         # Create tables
         engine = db.get_engine()
         Base.metadata.create_all(engine)
-        # Write dependencies to pre-processing dependency table
+        # Get dependencies for pre-processing dependency table
+        reversed_downstream_dependents = []
+        for dep in downstream_dependents:
+            if dep.reserve_direction is True:
+                reversed_downstream_dependents.append(dep.reverse_direction())
```

Comment:
Should the "reversed" dependencies be added to upstream dependencies in the dependency_config module instead of here?
```suggestion
        # Get dependencies for pre-processing dependency table
        reversed_downstream_dependents = []
        for dep in downstream_dependents:
            if dep.reverse_direction:
                reversed_downstream_dependents.append(dep.reverse_direction())
```

Reply:
Yeah, that makes sense.